css/style.css

```diff
@@ -6,3 +6,73 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+* {
+  font-family: Century Gothic;
+}
+
+header {
+  font-size: 35%;
+  text-align: center;
+  background-color: #f6f6f7;
+  width: 100%;
+  padding: 7px;
+  margin-bottom: 3rem;
+  border-radius: 3rem;
+}
+
+h1 {
+  font-style: bold;
+  font-size: 5rem;
+  color: #9160c3;
+  font-size: 30em;
+  margin-bottom: 2em;
+}
+
+header h2 {
+  font-style: ;
+  color: #a595b5;
+  font-size: 30em;
+  margin-bottom: 1em;
+}
+.introduction {
+  font-size: 3rem;
+  position: sticky;
+  margin-bottom: 3rem;
+  margin: 7rem;
+}
+.introduction image {
+  position: center;
+  border-radius: 3rem;
+}
```

Comment:
You need to use `.introduction img` to select the `img` tag. You might also want to shrink the image, for example you could use `width: 50vw;` to make it 50% of the browser width. Otherwise it is too large so it won't actually position itself in the centre.